dmscripts/notify_suppliers_of_new_questions_answers.py

```diff
@@ -130,6 +130,9 @@ def main(data_api_url, data_api_token, email_api_key, stage, dry_run, supplier_i
     logger.info(
         "{} suppliers found interested in these briefs".format(len(interested_suppliers))
     )
+    logger.info(
+        "Supplier IDs found: {}".format(",".join(map(str, interested_suppliers.keys())))
+    )
```

Comment:
Can we combine the two log statements above?

Reply:
Also let's check it's fine in Python3 (I think it will be fine...)

Reply:
Doesn't Travis check for Python3 compatibility?

Reply:
Travis will check the unit tests in python 3 but won't actually do a functional run of the script so that is what is missing. I have tested a dry run on production using python 3 which should be sufficient for the moment I think.